activities/GetThingsDone.activity/js/activity.js

```diff
@@ -1,4 +1,4 @@
-define(["sugar-web/activity/activity","sugar-web/datastore","activity/model","activity/view","activity/controller"], function (activity,datastore,model,view,controller) {
+define(["sugar-web/activity/activity", "sugar-web/datastore", "activity/model", "activity/view", "activity/controller"], "l10n", function (activity, webL10n, datastore, model, view, controller) {
```

Comment:
Changing this line is not needed - we don't have to import `webL10n`, we have already declared global translation interface in `js/loader.js`.